.betterer.results

```diff
@@ -3038,6 +3038,9 @@ exports[`better eslint`] = {
       [0, 0, 0, "Do not use any type assertions.", "0"],
       [0, 0, 0, "Do not use any type assertions.", "1"]
     ],
+    "public/app/features/correlations/transformations.ts:5381": [
+      [0, 0, 0, "Do not use any type assertions.", "0"]
```

Comment:
opened https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64263 to get rid of this